deployments/warp_routes/ETH/basecamp-sepolia-deploy.yaml

```
@@ -0,0 +1,10 @@
+basecamp:
+  interchainSecurityModule: "0x0000000000000000000000000000000000000000"
+  mailbox: "0xF53f7866775f6F0ACB81181D262a7E9559E9584E"
+  owner: "0x5CFADb51c6306e344dC69E5D7f05b02a4eC8bb9D"
+  type: native
+sepolia:
+  interchainSecurityModule: "0x0000000000000000000000000000000000000000"
+  mailbox: "0xcc737a94fecaec165abcf12ded095bb13c2ec30b"
+  owner: "0x5CFADb51c6306e344dC69E5D7f05b02a4eC8bb9D"
+  type: synthetic 
```

think you can get away without the ISM 0x0
```suggestion
basecamp:
  mailbox: "0xF53f7866775f6F0ACB81181D262a7E9559E9584E"
  owner: "0x5CFADb51c6306e344dC69E5D7f05b02a4eC8bb9D"
  type: native
sepolia:
  mailbox: "0xcc737a94fecaec165abcf12ded095bb13c2ec30b"
  owner: "0x5CFADb51c6306e344dC69E5D7f05b02a4eC8bb9D"
  type: synthetic 
```